views/postgres/index.erb

```diff
@@ -35,12 +35,16 @@
   <div class="grid gap-6">
     <%== part(
       "components/table_card",
-      headers: ["Name", "Managed by", "Location", "State"],
+      headers: ["Name", "Compute", "Storage", "Version", "Type", "Managed by", "Location", "State"],
       rows: @postgres_databases.map do |pg|
         brand, = flavors[pg[:flavor]]
         [
           [
             [pg[:name], {link: @project_data[:path] + pg[:path]}],
+            [pg[:vm_size]],
+            [pg[:storage_size_gib]],
+            [pg[:version]],
+            [pg[:read_replica] ? "Read Replica" : "Primary"],
             ["<div class='inline-flex gap-x-1 items-center'><img src='/icons/pg-#{brand.downcase}.png' class='object-cover rounded-lg overflow-hidden w-6 h-6'/><p>#{brand}</p></div>", {escape: false}],
```

Comment:
How does it look if we indent read replicas few pixels? If it looks good, we might remove read replica column.

Reply:
I have solved it by changing the column to "parent" and then ordering the replicas under the parent